src/Gui/decompiler-menus.xml

```diff
@@ -80,6 +80,7 @@
 	<cmd id="FileOpen" cmdSet="Reko" container="FileOpen" image="0">_Open...</cmd>
   <cmd id="FileOpenAs" cmdSet="Reko" container="FileOpenAs">Op_en As...</cmd>
 	<cmd id="FileSave" cmdSet="Reko" container="FileSave" image="1">_Save</cmd>
+  <cmd id="FileNewScript" cmdSet="Reko" container="FileNewScript">New script file...</cmd>
```

Comment:
Comment: I'm fine with the naming here, for the time being. Once we start adding lots of new file categories, it may be advisable to create a nested menu  `Add new` >`'Script file / Assembler file /... ` -- but not now.